packages/components/src/DateTimePickers/InputTimePicker/InputTimePicker.component.js

```diff
@@ -56,45 +62,60 @@ export default function InputTimePicker(props) {
 					{({ ref, style }) => (
 						<div id={popoverId} className={theme.popper} style={style} ref={ref}>
 							<TimePicker
+								textInput={datetime.timeTextInput}
 								useSeconds={props.useSeconds}
-								onChange={(event, payload) =>
-									handlers.onChange(event, { ...payload, origin: 'PICKER' }, inputRef.current)
-								}
+								onChange={timePickerManagement.onChange}
 							/>
 						</div>
 					)}
 				</Popper>
 		),
 	].filter(Boolean);
 	return (
-		<DateTime.Manager
-			selectedTime={props.selectedTime}
-			useSeconds={props.useSeconds}
-			onChange={(...args) => handlers.onChange(...args, inputRef.current)}
+		<FocusManager
+			divRef={containerRef}
+			onFocusIn={handlers.onFocus}
+			onFocusOut={handlers.onBlur}
+			onKeyDown={event => {
+				handlers.onKeyDown(event, inputRef.current);
+			}}
 		>
-			<FocusManager
-				divRef={containerRef}
-				onFocusIn={handlers.onFocus}
-				onFocusOut={handlers.onBlur}
-				onKeyDown={event => {
-					handlers.onKeyDown(event, inputRef.current);
-				}}
-			>
-				{timePicker}
-			</FocusManager>
-		</DateTime.Manager>
+			{timePicker}
+		</FocusManager>
 	);
 }
 
 InputTimePicker.displayName = 'InputTimePicker';
 
 InputTimePicker.propTypes = {
 	id: PropTypes.string.isRequired,
-	selectedTime: PropTypes.string,
 	useSeconds: PropTypes.bool,
 	onChange: PropTypes.func,
 	onBlur: PropTypes.func,
 };
+
 InputTimePicker.defaultProps = {
 	useSeconds: false,
 };
+
+export default function ContexualInputTimePicker(props) {
```

Comment:
This is for using InputTimePicker standalone, without date.

Reply:
You can merge this component with InputTimePicker.
InputTimePicker won't be working alone. So there is no need to have it. What we expect in an InputTimePicker is that the input and the picker work together. So we need the manager in it

Reply:
I will need InputTimePicker without Time.Manager in InputDateTimePicker(DateTime.Manager will be used there)

Reply:
ok I see.